src/component/tooltip/TooltipContent.js

```diff
@@ -291,15 +291,13 @@ TooltipContent.prototype = {
         if (document.defaultView && document.defaultView.getComputedStyle) {
             var stl = document.defaultView.getComputedStyle(this.el);
             if (stl) {
-                width += parseInt(stl.paddingLeft, 10) + parseInt(stl.paddingRight, 10)
-                    + parseInt(stl.borderLeftWidth, 10) + parseInt(stl.borderRightWidth, 10);
-                height += parseInt(stl.paddingTop, 10) + parseInt(stl.paddingBottom, 10)
-                    + parseInt(stl.borderTopWidth, 10) + parseInt(stl.borderBottomWidth, 10);
+                width += parseInt(stl.borderLeftWidth, 10) + parseInt(stl.borderRightWidth, 10);
+                height += parseInt(stl.borderTopWidth, 10) + parseInt(stl.borderBottomWidth, 10);
             }
```

Comment:
It probably depends on the `box-sizing` of the current page.
Should we detect `box-sizing` to determine whether add padding here?
@Ovilia @pissang

Reply:
I think no matter what the `box-sizing` is, the `clientHeight` always includes padding.

Reply:
@oustn 's correct. Made a demo at https://jsfiddle.net/ovilia/hx04kzu1/7/ . With or without `box-sizing: border-box;`, `clientWidth` contains padding all the same.